tests/unit/utils/txn_parser/test_get_order_book_changes.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import json
+from unittest import TestCase
+
+from xrpl.utils import get_order_book_changes
+
+path_to_json = "tests/unit/utils/txn_parser/transaction_jsons/"
+with open(path_to_json + "offer_created.json", "r") as infile:
+    offer_created = json.load(infile)
+with open(path_to_json + "offer_partially-filled_and_filled.json", "r") as infile:
+    offer_partially_filled_and_filled = json.load(infile)
+with open(path_to_json + "offer_cancelled.json", "r") as infile:
+    offer_cancelled = json.load(infile)
+with open(path_to_json + "offer_with_expiration.json", "r") as infile:
+    offer_with_expiration = json.load(infile)
+
+
+class TestGetOrderBookChanges(TestCase):
+    def test_offer_created(self: TestGetOrderBookChanges):
+        actual = get_order_book_changes(offer_created["meta"])
+        expected = [
+            {
+                "account": "rJHbqhp9Sea4f43RoUanrDE1gW9MymTLp9",
+                "offer_changes": [
+                    {
+                        "flags": 131072,
+                        "taker_gets": {"currency": "XRP", "value": "44.930000"},
+                        "taker_pays": {
+                            "currency": "USD",
+                            "issuer": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
+                            "value": "14.524821",
+                        },
+                        "sequence": 71307620,
+                        "status": "created",
+                        "maker_exchange_rate": "0.323276674827509",
+                        "expiration_time": 740218424,
+                    }
+                ],
+            }
+        ]
+        self.assertEqual(actual, expected)
+
+    def test_offer_partially_filled_and_filled(self: TestGetOrderBookChanges):
+        actual = get_order_book_changes(offer_partially_filled_and_filled["meta"])
+        expected = [
+            {
+                "account": "rNzgS71DyJPMnWMA8aS7NqvXP7bNuwyaZo",
+                "offer_changes": [
+                    {
+                        "flags": 131072,
+                        "taker_gets": {
+                            "currency": "USD",
+                            "issuer": "rhub8VRN55s94qWKDv6jmDy1pUykJzF3wq",
+                            "value": "-63.7479881398749",
+                        },
+                        "taker_pays": {
+                            "currency": "USD",
+                            "issuer": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
+                            "value": "-62.4730283770749",
+                        },
+                        "sequence": 5931,
+                        "status": "filled",
+                        "maker_exchange_rate": "0.979999999999960",
+                    }
+                ],
+            },
+            {
+                "account": "rPu2feBaViWGmWJhvaF5yLocTVD8FUxd2A",
```

Comment:
This account is always the maker since these changes are recording only changes to the ledger object right? (If so it might make sense to call this field `maker_account`)

Reply:
Correct `account` is always the owner of the  account. `maker_account` would be a way clearer name 👍